src/index.js

```diff
@@ -44,6 +44,14 @@ class Dropzone extends React.Component {
   onDragEnter(e) {
     e.preventDefault();
 
+    const dt = e.dataTransfer;
+
+    // If the object isn't a file (dom element, link, etc.)
+    if (!(dt.types && (dt.types.indexOf ? dt.types.indexOf('Files') !== -1 : dt.types.contains('Files')))) {
```

Comment:
`.contains` isn't implemented in most of the browsers yet AFAIK. I'd rather had this extracted to a util function with separate tests.